lib/iris/tests/unit/cube/test_Cube.py

```diff
@@ -1794,5 +1795,21 @@ def test_preserves_lazy(self):
         self.assertArrayAllClose(cube.data, real_data_ft)
 
 
+@tests.skip_data
+class Test__repr_html(tests.IrisTest):
+    def test_bad_repr(self):
+        # If an html repr fails we don't want it to pass the error.
+        cube = stock.simple_3d()
+        bad_attr = {'history': 'First edit\nSecond edit'}
+        cube.attributes.update(bad_attr)
+        representer = CubeRepresentation(cube)
+
+        # Confirm we get an exception when we make the html repr...
+        with self.assertRaisesRegexp(ValueError, 'substring'):
```

Comment:
This will stop erroring once #3373 is merged, so we will have to find / cause a different exception in the representer instead...